### PR TITLE
Zoey_Completed css-layout-classes

### PR DESCRIPTION
**Pull request by Zoey1318**
css-layout-classes practice

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>css-layout-classes practice</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
```
Comment (scrumptiousdev) [inline css-layout-classes/index.html:13]: Are we missing `.column-full` styling in CSS?

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Nicely done! Just one super tiny change request!

**Comment by Zoey1318** [comment]
css-layout-classes practice2

**Review by scrumptiousdev — APPROVED** [review]
🪄 Approved!

**Pull request merged**